onnx/__init__.py

```diff
@@ -14,6 +14,60 @@
 import google.protobuf.message  # type: ignore
 
 
+def _load_string(obj):
+    if hasattr(obj, 'read') and callable(obj.read):
+        s = obj.read()
+    else:
+        with open(obj, 'rb') as f:
+            s = f.read()
+    return s
+
+
+def _save_string(str, f):
+    if hasattr(f, 'write') and callable(f.write):
+        f.write(obj, f)
```

Comment:
str

Reply:
err... i ran the wrong scripts to test this ;-)